src/log.cpp

```diff
@@ -95,8 +101,7 @@ void Log::log(LogLevel level, const char* msg, va_list args) {
     }
     buf[prefix_len + msg_len] = '\n';
 
-    // Write all messages to JFR, if active
-    if (level < LOG_ERROR) {
+    if (level >= FlightRecorder::MIN_LOG_LEVEL) {
```

Comment:
Have you removed `< LOG_ERROR` intentionally?
Error message typically indicates a situation when profiler is unable to execute a user command. Errors don't happen out of the blue at runtime and don't need to be recorded.

Reply:
Got it, 148e6b3e46bb8699e32baa831a11101f6740809f